<launch>

	<node pkg="sensors_processing" type="robot_move" name="robot_move" output="screen">
		<rosparam file="$(find elektron_navigation)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />	
		<rosparam file="$(find elektron_navigation)/config/local_costmap_params.yaml" command="load" />	
	</node>

</launch>
